dashboard/common/db_util.py

```diff
@@ -87,24 +88,24 @@ def get_default_user_course_id(user_id):
 
 def get_user_courses_info(username):
```

Comment:
@pushyamig, can you take a look at this alternative approach I helped with? I know you are composing `myla_globals` differently in your code, putting `lti_`-specific items in there, but I guess I'm somewhat concerned that enrollment roles still need to be associated with a course, as I don't think we're disabling the ability for a user to navigate through the UI to different courses with MyLA enabled. How do you think we should combine this approach with what you are doing?

Reply:
Can we continue this conversation in the thread? It's nice to be able to collapse this later once it's resolved.

You are right regarding multiple roles for a course, I'll look into that; I think we should resolve it here. But yes, I do think we need to attach the role(s) at the course level, not globally for that user.